stories/menu.stories.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import { storiesOf } from '@storybook/react';
+
+/**
+ * Internal dependencies
+ */
+import Menu from '../assets/js/components/Menu';
+
+storiesOf( 'Global', module )
+	.add( 'Menu', () => {
+		return (
+			<div>
+				<div>
+					<p>Menu</p>
+					<Menu
+
+						menuOpen
+						menuItems={ [
+							'Menu 1',
+							'Menu 2',
+							'Menu 3',
+							'Menu 4',
+							'Menu 5',
+						] }
+						onSelected={ () => {
+							console.log( 'Selected' ); // eslint-disable-line
+						} }
```

Comment:
Same here: let's output selected element to console and use `global` to avoid eslint issues.

Reply:
Updated @eugene-manuilov